### PR TITLE
Update azure-virtual-desktop-multi-session.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about AADDS

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger9** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change. @smbhardwaj

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2677

File: memdocs/intune/fundamentals/azure-virtual-desktop-multi-session.md [hunk 1]
```diff
@@ -59,6 +60,9 @@ This feature supports Windows 10 or Windows 11 Enterprise multi-session VMs whic
   - [Configuration Manager co-management](/configmgr/comanage/overview).
 - Azure AD-joined and enrolled in Microsoft Intune by enabling [Enroll the VM with Intune](/azure/virtual-desktop/deploy-azure-ad-joined-vm#deploy-azure-ad-joined-vms) in the Azure portal.
 
+> [!Note]
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/azure-virtual-desktop-multi-session.md:63]:
```suggestion
> [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger12** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change. @smbhardwaj

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@SMBhardwaj Please review and signoff this PR. Thanks

**Comment by PRMerger8** [comment]
@Smritib17 : Thanks for your contribution! The author(s) have been notified to review your proposed change. @smbhardwaj

**Review by Smritib17 — APPROVED** [review]

**Pull request merged**